docs/changes/coordinates/14379.bugfix.rst

```diff
@@ -0,0 +1 @@
+The bug is fixed in ``to_string``.
```

Comment:
You would have to elaborate what "the bug" is here. This will be part of the change log for the whole package.